pkg/snapshot/clonetree_net.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 const (
+	// warning: don't use the context package here, this means not even the linuxpath package.
+	// TODO(fromani) remove the path duplication
```

Comment:
Is the above still true now the you've split pci address into its own package? Why don't we want to use the context package here?

Reply:
thing is: context wants to manage snapshots, to transparently consume them. And this is by design, and we should keep it. So if we import `linuxpath` here... 
```
go vet 
package github.com/jaypipes/ghw
	imports github.com/jaypipes/ghw/pkg/baseboard
	imports github.com/jaypipes/ghw/pkg/context
	imports github.com/jaypipes/ghw/pkg/snapshot
	imports github.com/jaypipes/ghw/pkg/linuxpath
	imports github.com/jaypipes/ghw/pkg/context: import cycle not allowed
```
obviously not very different importing `context`, just for the record:
```
go vet 
package github.com/jaypipes/ghw
	imports github.com/jaypipes/ghw/pkg/baseboard
	imports github.com/jaypipes/ghw/pkg/context
	imports github.com/jaypipes/ghw/pkg/snapshot
	imports github.com/jaypipes/ghw/pkg/context: import cycle not allowed
```
There is however another way out: we can split the `clonetree` code (which needs to walk through the paths) code from the `snapshot` package, or we can split the `snapshot package` in a simpler subpackage which consumes snapshot, and which `context` will import, and in another, more complex and richer package which will create snapshots and which will consume (not be consumer of) `context`.

I don't have strong preferences here and each approach has pros and cons, so, I'd be happy to fo the direction you like most.